### PR TITLE
Update CHANGES.md for v1.0.16

### PR DESCRIPTION
**Pull request by stealthcode**

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
:+1:

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
 
+### Version 1.0.16 – November 11 2015 ([Maven Central](http://search.maven.org/#artifactdetails%7Cio.reactivex%7Crxjava%7C1.0.16%7C)) ###
+* [Pull 3169] (https://github.com/ReactiveX/RxJava/pull/3169) Merge can now operate in horizontally unbounded mode
+* [Pull 3286] (https://github.com/ReactiveX/RxJava/pull/3286) Implements BlockingSingle
+* [Pull 3433] (https://github.com/ReactiveX/RxJava/pull/3433) Add Single.defer()
+* [Pull 3468] (https://github.com/ReactiveX/RxJava/pull/3468) Fix other places that may swallow OnErrorFailedException
+* [Pull 3485] (https://github.com/ReactiveX/RxJava/pull/3485) fix scan() not accepting a null initial value
+* [Pull 3488] (https://github.com/ReactiveX/RxJava/pull/3488) Remove all instances of Atomic*FieldUpdater
+* [Pull 3510] (https://github.com/ReactiveX/RxJava/pull/3510) eager concatMap to choose safe or unsafe queue based on platform
+* [Pull 3511] (https://github.com/ReactiveX/RxJava/pull/3511) Remove unused and inefficient SubscriptionRandomList
```
Comment (JakeWharton) [inline CHANGES.md]: This wasn't public API. Shouldn't need to mention.

**Comment by stealthcode** [comment]
@abersnaze @benjchristensen @akarnokd @zsxwing @stevegury

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 # RxJava Releases #
 
+### Version 1.0.16 – November 11 2015 ([Maven Central](http://search.maven.org/#artifactdetails%7Cio.reactivex%7Crxjava%7C1.0.16%7C)) ###
+* [Pull 3169] (https://github.com/ReactiveX/RxJava/pull/3169) Merge can now operate in horizontally unbounded mode
+* [Pull 3286] (https://github.com/ReactiveX/RxJava/pull/3286) Implements BlockingSingle
+* [Pull 3433] (https://github.com/ReactiveX/RxJava/pull/3433) Add Single.defer()
+* [Pull 3468] (https://github.com/ReactiveX/RxJava/pull/3468) Fix other places that may swallow OnErrorFailedException
+* [Pull 3485] (https://github.com/ReactiveX/RxJava/pull/3485) fix scan() not accepting a null initial value
+* [Pull 3488] (https://github.com/ReactiveX/RxJava/pull/3488) Remove all instances of Atomic*FieldUpdater
```
Comment (JakeWharton) [inline CHANGES.md]: Should probably read "Replace all instances of Atomic*FieldUpdater with direct Atomic* instances" or something.

Comment (stealthcode) [inline CHANGES.md]: Thanks

**Comment by akarnokd** [comment]
Can we get #3493 in as well?

**Comment by stealthcode** [comment]
Merged #3493 and updated.

File: CHANGES.md (hunk 2 above)
Comment (artem-zinnatullin) [inline CHANGES.md]: Well, it was internal change too :) If we want to show what this PR means to the users — we can write something like: "Replace … (fixes crashes on some Samsung devices Running on Android 5)".

**Comment by akarnokd** [comment]
:+1:

**Comment by abersnaze** [comment]
LGTM :+1:

**Comment by stealthcode** [comment]
Thanks @akarnokd @abersnaze @JakeWharton @artem-zinnatullin

**Comment by JakeWharton** [comment]
No thanks to you maintainers for pushing these along!

**Pull request merged**